src/Symfony/Component/Validator/Resources/translations/validators.fr.xlf

```diff
@@ -334,6 +334,10 @@
                 <source>This value should be valid JSON.</source>
                 <target>Cette valeur doit être un JSON valide.</target>
             </trans-unit>
+            <trans-unit id="87">
```

Comment:
This id should be `91` (translations from #28637 #30900 are missing too).

Reply:
it's `87` everywhere else.

Reply:
Fixed